specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Batch Document Translation API",
```

Comment:
Title will be used as client name for generated SDK, so here better be "*** Client"